miden/Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "miden"
 path = "src/main.rs"
 bench = false
 doctest = false
-required-features = ["executable"]
+required-features = ["executable", "internal"]
```

Comment:
Instead of adding `internal` feature here, I'd probably add it as dependency feature for the `executable` feature.

Reply:
good point, done